ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/vmgenv1/cucushift-installer-rehearse-azure-ipi-vmgenv1-workflow.yaml

```diff
@@ -0,0 +1,16 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-vmgenv1
+  steps: 
+    pre:
+    - chain: cucushift-installer-rehearse-azure-ipi-vmgenv1-provision
+    post:
+    - chain: cucushift-installer-rehearse-azure-ipi-vmgenv1-deprovision
+  documentation: |-
```

Comment:
Pls add some description to say this testing is meaningful only for 4.11, later versions